ui/plugins/controller.js

```diff
@@ -157,7 +157,7 @@ treeherder.controller('PluginCtrl', [
                     // Provide a parse status for the model
                     var logsNotParsed = [];
                     $scope.jobLogsAllParsed = _.every($scope.job_log_urls, function(jlu) {
-                        if(jlu.parse_status === 'pending'){
+                        if(jlu.parse_status === 'pending' || jlu.parse_status === 'failed'){
```

Comment:
We can discard this change.  I apologize I initially confirmed this was the right path.  See my last comment on the overall PR.